source/wp-content/themes/wporg-parent-2021/sass/base/_layout.scss

```diff
@@ -26,3 +26,76 @@
 		max-width: var(--wp--custom--alignment--aligned-max-width);
 	}
 }
+
+// Configures a 3 column layout where `wporg/sidebar-container` blocks sit either side of the main content.
+// Examples: Developer Resources, Documentation.
+.has-three-columns {
```

Comment:
Considered putting this in a new `blocks/_group.scss` partial and scoping it to `.wp-block-group`, but I'm not sure if there is any benefit.

Reply:
Keeping it in `_layout` makes sense to me 👍🏻